doc/src/templates/landing-main/contrib-section/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import Card from '../../../../../src/card';
+import Button from '../../../../../src/button';
+import engineerImage from '../../../images/engineer.svg';
+import developerImage from '../../../images/man-in-office-desk-with-computer.svg';
+import contributeImage from '../../../images/triple-arrow-merging-to-one.svg';
+import theme from './index.scss';
+import FooterSection from './footer';
+
+class ContributionSection extends Component {
+  render() {
```

Comment:
Can we use functional component here?
Since we have nothing to do with the state.